src/main/webapp/dev/js/common/questionMsq.js

```diff
@@ -130,7 +132,23 @@ function addMsqOption(questionNum) {
             </span>
         </div>
     </div>
-    `).insertBefore($(`#msqAddOptionRow-${questionNum}`));
+    `);
+
+    const weightFragment = (`
+    <div class="margin-bottom-7px">
+        <input type="number" class="form-control nonDestructive" value="0"
+                id="${ParamsNames.FEEDBACK_QUESTION_MSQ_WEIGHT}-${curNumberOfChoiceCreated}-${questionNum}"
+                name="${ParamsNames.FEEDBACK_QUESTION_MSQ_WEIGHT}-${curNumberOfChoiceCreated}" step="0.01" min="0" required>
```

Comment:
Newly added option's corresponding weight box may not be `required` if weight feature is not enabled.

Reply:
Same for MCQ question?

Reply:
Checked in Chrome. Chrome won't check required for hidden field. Not a big deal :)